src/Application/GameMenu.cpp

```diff
@@ -33,25 +33,36 @@ using Io::TextInputType;
 using Io::KeyToggleType;
 using Io::InputAction;
 
+enum class CurrentConfirmationState {
+    CONFIRM_NONE,
+    CONFIRM_NEW_GAME,
+    CONFIRM_QUIT
+};
+using enum CurrentConfirmationState;
+
+bool initialLoadSlotSelection = false;
```

Comment:
That's a very misleading variable name.

`initialLoadSlotSelected` would mean that we have already selected a slot.
`initialLoadSlotSelectionPending` would mean that we're waiting for initial selection.

`initialLoadSlotSelection` kinda means both, depending on who's reading X).

Let's come up with a better name and add a short doc.